packages/backend-for-frontend/src/routers/inAppNotificationRouter.ts

```diff
@@ -0,0 +1,32 @@
+import { bffApi } from "pagopa-interop-api-clients";
+import { ZodiosRouter } from "@zodios/express";
+import {
+  ZodiosContext,
+  ExpressContext,
+  zodiosValidationErrorToApiProblem,
+} from "pagopa-interop-commons";
+import { ZodiosEndpointDefinitions } from "@zodios/core";
+
+const inAppNotificationRouter = (
+  ctx: ZodiosContext
+): ZodiosRouter<ZodiosEndpointDefinitions, ExpressContext> => {
+  const inAppNotificationRouter = ctx.router(bffApi.inAppNotificationsApi.api, {
+    validationErrorHandler: zodiosValidationErrorToApiProblem,
+  });
+
+  inAppNotificationRouter
+    .get("/in-app-notifications", (_, res) => res.status(501).send())
+    .post("/in-app-notifications/bulk/markAsRead", (_, res) =>
+      res.status(501).send()
+    )
+    .post("/in-app-notifications/:notificationId/markAsRead", (_, res) =>
+      res.status(501).send()
+    )
+    .delete("/in-app-notifications/:notificationId", (_, res) =>
```

Comment:
In API V2, we agreed that the standard for APIs from now on should be `camelCase` instead of `kebab-case`:
https://pagopaspa.slack.com/archives/C08J74Y0H96/p1750932376948449

So maybe here we should use `inAppNotifications`? No strong opinion, if you have already used this standard in other notification-related services let's keep it as it is...
but I would track somewhere to align all these to the new standard before deploying the notification stuff